elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/model/Table.java

```diff
@@ -111,11 +111,9 @@ public String getDescription() {
 
     public enum Cardinality {
 
-        TINY("tiny"),
         SMALL("small"),
         MEDIUM("medium"),
-        LARGE("large"),
-        HUGE("huge");
+        LARGE("large");
```

Comment:
We don't want to remove labels here.  We want to add TINY and HUGE to: com.yahoo.elide.datastores.aggregation.annotation.Cardinality.